enclave/core/atexit.c

```diff
@@ -43,8 +44,8 @@ static oe_atexit_entry_t* _new_atexit_entry(void (*func)(void*), void* arg)
 {
     oe_atexit_entry_t* entry;
 
-    if ((entry = (oe_atexit_entry_t*)oe_sbrk(sizeof(oe_atexit_entry_t))) ==
-        (void*)-1)
+    if ((entry = (oe_atexit_entry_t*)oe_nodebug_malloc(
```

Comment:
Nit: can you correct the comment on line 38 since it no longer uses `sbrk()` for this purpose?

Reply:
Sure, done.